oak-lucene/src/test/java/org/apache/jackrabbit/oak/composite/blueGreen/CustomizedIndexTest.java

```diff
@@ -211,4 +233,25 @@ private void createFolders() throws IOException {
         indexDir = tempDir.newFolder("index");
     }
 
+    private static String getActiveLuceneIndexes(Persistence p) {
+        return getActiveLuceneIndexes(p.getCompositeNodestore(), p.getMountInfoProvider());
+    }
+
+    private static String getActiveLuceneIndexes(NodeStore ns, MountInfoProvider m) {
+        ArrayList<String> list = new ArrayList<>();
+        IndexInfoServiceImpl indexService = new IndexInfoServiceImpl(ns,
+                new IndexPathServiceImpl(ns, m));
+        for (IndexInfo info : indexService.getAllIndexInfo()) {
+            if (!LuceneIndexConstants.TYPE_LUCENE.equals(info.getType())) {
+                continue;
+            }
+            if (!info.isActive()) {
+                continue;
+            }
+            list.add(info.getIndexPath());
+        }
+        Collections.sort(list);
+        return list.toString();
+    }
```

Comment:
I find the name of this method confusing. From the name, without looking at the return type, I expected that it returns a list of the active indexes. But in fact, this converts to String description, so it should have a name like toString().

Why are you doing the test assertions on a string representation of the list of the indexes instead of using Java lists or Java sets? I think using lists or sets is less error prone and more clear, because conceptually that is what we are comparing, not strings.

Reply:
Why would it be less error prone?

Reply:
Whether lists of strings are compared or string representations of lists of strings... I don't see what difference it would make... But OK, I'll change it.

Reply:
If we use strings, we are comparing two things:
- that the index names are the same (the goal of the test)
- that the String format used to convert a list of names to a single string is the same in the expected and actual values

We care only about the index names being the same, the string format is an additional complexity that we are introducing only for the test. For instance, we need to get the commas and the spacing right, while if we compare based on `Set<String>` we avoid these details. 

Why would it be easier to fix the tests with Strings as compared with having something like ` assertEquals(Set.of("/oak:index/lucene", "/oak:index/test-2")`, ...`

Anyway, it's not a big issue.

Reply:
With a single String, my IDE allows to copy & paste... I'll try if that works with sets as well.